src/site/antora/modules/ROOT/nav.adoc

```diff
@@ -40,20 +40,23 @@
 ** xref:manual/thread-context.adoc[]
 ** xref:manual/scoped-context.adoc[]
 ** xref:manual/resource-logger.adoc[]
-* xref:manual/configuration.adoc[]
+* Backend Configuration
+** xref:manual/configuration.adoc[Configuration file]
+** xref:manual/appenders.adoc[]
+** xref:manual/layouts.adoc[]
+*** xref:manual/json-template-layout.adoc[]
+** xref:manual/lookups.adoc[]
+** xref:manual/customloglevels.adoc[]
```

Comment:
They can be configured through the API or a configuration file.

Reply:
My point was, you called this navigation group _"**Backend** configuration"_, yet it contains an API-related entry. But since you renamed it back to _"Configuration"_, it should be good now.